CenterForDigitalHumanities/TPEN-services
# Test User Class

Testing procedure

Go to https://user.t-pen.org and log in to get the Bearer Token.  Make sure there is no tpen.user in your mongo with an `agent` property that matches in the `agent` in the token.

npm start

Make an authenticated request to http://localhost:3009/user/tester

![image](https://github.com/CenterForDigitalHumanities/TPEN-services/assets/3287006/29cf1b82-e946-4199-aac5-749559177884)

Check logs for info.

The desired result is a tpen.users user whose `_id` matches the hash from the Agent URI.
